modules/infrastructure/organizational/cloudvision-role/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 0.15.0"
+  required_providers {
+    aws = {
+      version               = ">= 3.50.0"
+      configuration_aliases = [aws.member]
```

Comment:
bumped terraform 0.14.0 to 0.15.0 to enable alias ussage for multiple-provider modules